.github/workflows/php-library-static.yml

```diff
@@ -0,0 +1,75 @@
+name: Static analysis
+
+on:
+ workflow_call:
+ inputs:
+    php-versions:
+     required: true
+     type: string
+jobs:
+  phpstan:
+   
+    name: PHPStan 
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        php: ${{ fromJSON(inputs.php-versions) }}
```

Comment:
PHP versions do not affect static analysis tools. 

May be better to remove the PHP version input from the input/matrix.